Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj

```diff
@@ -50,10 +50,6 @@
     </Reference>
     <Reference Include="PX.Objects">
       <HintPath>..\..\..\..\Bin\PX.Objects.DLL</HintPath>
-    </Reference>
```

Comment:
@MehranQadri , I'm not sure why this reference is removed. It seems it should not be.